client/components/Profile.jsx

```diff
@@ -34,24 +38,51 @@ class Profile extends React.Component {
               <fieldset>
                 <legend>Profile</legend>
 
-                <label htmlFor='username'>Username</label>
+                {/* <img src={profilePic} alt={name}> </img> */}
+
+                <label htmlFor='username'>Username: </label>
                 <input id='username' name='username' placeholder='username'
                   onChange={this.handleChange} value={username} />
+                <br/>
```

Comment:
Let's remove all of these line breaks. Let's use CSS styling for formatting instead.

Reply:
Yes boss